base/kra/src/main/java/com/netscape/kra/RecoveryService.java

```diff
@@ -629,6 +629,13 @@ public void createPFX(IRequest request, Hashtable<String, Object> params,
                     logger.debug("RecoverService: createPFX() key not null");
                 }
             } else {
+	        String nonLegacyAlg = engine.getConfig().getString("kra.nonLegacyAlg", "AES/CBC/NoPadding");
```

Comment:
Should add comment in kra's CS.cfg to tell people what options they have here for kra.nonLegacyAlg